File: Beginner/1435 - Square Matrix I/1435.c
#include <stdio.h>

int main() {
    int N;
    scanf("%d", &N);

    int X[N];
    for (int i = 0; i < N; i++) {
        scanf("%d", &X[i]);
    }

    int R = X[0];
    int pos = 0;

    for (int i = 1; i < N; i++) {
        if (X[i] < R) {
            R = X[i];
            pos = i;
        }
    }

    printf("Menor valor: %d\n", R);
    printf("Posicao: %d\n", pos);

    return 0;
}

File: Beginner/1435 - Square Matrix I/1435.cs
using System;

class URI {
    static void Main() {
        int N = int.Parse(Console.ReadLine());
        string[] inputs = Console.ReadLine().Split(' ');

        int[] X = new int[N];
        for (int i = 0; i < N; i++) {
            X[i] = int.Parse(inputs[i]);
        }

        int R = X[0];
        int pos = 0;

        for (int i = 1; i < N; i++) {
            if (X[i] < R) {
                R = X[i];
                pos = i;
            }
        }

        Console.WriteLine($"Menor valor: {R}");
        Console.WriteLine($"Posicao: {pos}");
    }
}